wf/ja-jp/Content/Topics/Workflows/build/build-delegated-flow.htm

```diff
@@ -179,9 +141,9 @@
                                     <div class="nav-search row">
                                         <form class="search" action="#">
                                             <div class="search-bar search-bar-container needs-pie">
-                                                <input class="search-field needs-pie" type="search" aria-label="検索フィールド" placeholder="検索" />
-                                                <div class="search-filter-wrapper"><span class="invisible-label" id="search-filters-label">フィルタ:</span>
-                                                    <div class="search-filter" aria-haspopup="true" aria-controls="sf-content" aria-expanded="false" aria-label="検索フィルタ" title="すべてのファイル" role="button" tabindex="0">
+                                                <input class="search-field needs-pie" type="search" aria-label="Search Field（検索フィールド）" placeholder="検索" />
```

Comment:
same here. We got English back (although in this case it seems ok since we have Japanese translations in brackets)

Reply:
Yes, that change is not great, but it isn't exposed through the UI -- it's the Flare search widget, which we are hiding